test_apps/smokeTests/testResults/cf_baseline.txt

```diff
@@ -60,4 +60,4 @@ Time Coordinates:
 
 Time coordinate variable name: time
 Number of time steps: 1
-Elapsed time: 0.131529
+Elapsed time: 0.318985
```

Comment:
Should all of these *baseline* files contain the run time? Aren't they used for  comparison to test for regressions? You could add an option to disable them. I added an execution timer because I wanted to do performance evaluation, but I wasn't aware that the outputs were being used in this way.

Reply:
The final line in the baseline files that shows "Elapsed time" does not get compared in the diff.  I still need to scrub the new "Time" entries that you've added, but I think they're currently benign as all of our tests report a value of 0.  Still, these should be scrubbed in case they ever report non-zero.